crates/install-wheel-rs/src/linker.rs

```diff
@@ -548,6 +551,51 @@ fn hardlink_wheel_files(
     Ok(count)
 }
 
+/// Extract a wheel by symbolically-linking all of its files into site packages.
+fn symlink_wheel_files(
+    site_packages: impl AsRef<Path>,
+    wheel: impl AsRef<Path>,
+    locks: &Locks,
+) -> Result<usize, Error> {
+    let mut count = 0usize;
+
+    // Walk over the directory.
+    for entry in WalkDir::new(&wheel) {
+        let entry = entry?;
+        let path = entry.path();
+
+        let relative = path.strip_prefix(&wheel).unwrap();
+        let out_path = site_packages.as_ref().join(relative);
+
+        if entry.file_type().is_dir() {
+            fs::create_dir_all(&out_path)?;
+            continue;
+        }
+
+        // The `RECORD` file is modified during installation, so we copy it instead of symlinking.
+        if path.ends_with("RECORD") {
+            fs::copy(path, &out_path)?;
+            count += 1;
+            continue;
+        }
+
+        // Try to create a symlink, fallback to copying if it fails.
+        if let Err(err) = create_symlink(path, &out_path) {
```

Comment:
Do you mind adding the whole `Attempt::UseCopyFallback` thing from `hardlink_wheel_files`? It's helpful because if we detect that symlinks aren't supported, we basically stop trying.

Reply:
Done! Relegated this function to just being a copy of the hardlink function